variants/BharatPi-A7672S-4G/pins_arduino.h

```diff
@@ -0,0 +1,23 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+static const uint8_t LED_BUILTIN = 2;
+#define BUILTIN_LED LED_BUILTIN  // backward compatibility
+#define LED_BUILTIN LED_BUILTIN  // allow testing #ifdef LED_BUILTIN
+
```

Comment:
You are missing default SPI and I2C pins, please add them.
This must be done for all boards.
If you don't want to specify the default pin, you can use -1 as the pin so defaults from the core will be used.